src/themis/secure_message_wrapper.h

```diff
@@ -27,15 +27,14 @@
 #define THEMIS_SECURE_MESSAGE_RSA_SIGNED (THEMIS_SECURE_MESSAGE_SIGNED ^ 0x00000010)
 #define THEMIS_SECURE_MESSAGE_EC_SIGNED (THEMIS_SECURE_MESSAGE_SIGNED ^ 0x00000020)
 
-#define IS_THEMIS_SECURE_MESSAGE_SIGNED(tag) \
-    (((tag)&0xffffff00) == THEMIS_SECURE_MESSAGE_SIGNED ? true : false)
+#define IS_THEMIS_SECURE_MESSAGE_SIGNED(tag) (((tag)&0xffffff00) == THEMIS_SECURE_MESSAGE_SIGNED)
```

Comment:
I believe there is some arcane reason for using `? true : false` here – like C's boolean not being `bool` but something special – but I might also be wrong: `git blame` does not enlighten me, so I guess it's fine.

Reply:
isn't it a case when a two-bytes value `00100100 0000000` is casted to bool, leaving only the second byte `00000000` meaning `false`, white the first byte actually stored value (non-`false`).

Reply:
I like how this meaningless change makes everyone doubt in reality😂